pynamodb/connection/base.py

```diff
@@ -270,6 +277,18 @@ def dispatch(self, operation_name, operation_kwargs):
             log.debug("%s %s consumed %s units",  data.get(TABLE_NAME, ''), operation_name, capacity)
         return data
 
+    def send_post_boto_callback(self, operation_name, req_uuid, table_name):
+        try:
+            post_boto_send.send(self, operation_name=operation_name, table_name=table_name, req_uuid=req_uuid)
```

Comment:
i would remove boto from the signal names altogether. it's not unlikely that in the future that we will remove boto(core) entirely.